java/statestore/src/main/java/sleeper/statestore/transactionlog/DynamoDBTransactionLogStateStore.java

```diff
@@ -52,22 +55,32 @@ private static void loadLatestSnapshots(
         LatestSnapshots latestSnapshots = new DynamoDBTransactionLogSnapshotStore(instanceProperties, tableProperties, dynamoDB).getLatestSnapshots();
         TransactionLogSnapshotSerDe snapshotSerDe = new TransactionLogSnapshotSerDe(tableProperties.getSchema(), configuration);
         try {
-            latestSnapshots.getFilesSnapshot().ifPresent(filesSnapshot -> {
+            if (latestSnapshots.getFilesSnapshot().isPresent()) {
+                TransactionLogSnapshot filesSnapshot = latestSnapshots.getFilesSnapshot().get();
```

Comment:
Can we log the time taken to read the snapshot from S3 and deserialise it?

Reply:
Done, thanks